web/samples/uncompiled.html

```diff
@@ -46,6 +46,7 @@
     <script>
       var kmw=window.tavultesoft.keymanweb;
       kmw.init({
+		attachType:'auto',
```

Comment:
The latter - it really annoyed me when the page didn't work with touch platforms.  It's there as a testing page, so I figured it ought be enabled for all of 'em.  The new page makes a good master 'attachment' check, anyway.